src/Polly.RateLimiting.Tests/RateLimiterResilienceStrategyTests.cs

```diff
@@ -47,7 +47,7 @@ public void Execute_HappyPath()
     public void Execute_LeaseRejected(bool hasEvents, bool hasRetryAfter)
     {
         _diagnosticSource.Setup(v => v.IsEnabled("OnRateLimiterRejected")).Returns(true);
-        _diagnosticSource.Setup(v => v.Write("OnRateLimiterRejected", It.Is<object>(obj => obj is IResilienceArguments)));
+        _diagnosticSource.Setup(v => v.Write("OnRateLimiterRejected", It.Is<object>(obj => obj != null)));
```

Comment:
```suggestion
        _diagnosticSource.Setup(v => v.Write("OnRateLimiterRejected", It.Is<object>(obj => obj is not null)));
```

Reply:
It complains that expression tree cannot contain is operator. I have to leave it as it is.